src/components/About/styles.tsx

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
+
+export const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 960px;
+  margin-top: 400px;
+`;
+
+export const Box = styled.div`
+  border: 1px solid;
+  border-image: linear-gradient(
+    to right top,
+    rgb(0, 0, 0, 0) 40%,
```

Comment:
Só como sugestão, geralmente é bom apenas um arquivo com cores para manter  um controle maior, porém de toda forma está excelente, bom trabalho

Reply:
I agree. We do that on Flexa. So, when all the issues get finished, we can apply these "good practices" in another issue. I just used the pattern used previously.

Reply:
Awesome @brunosousadev. I think that a great path to this will be `src/constants`. We can create a file called `colors.ts` and inside this, create an `enum` contains the colors :smile:  
Anyway, you did that very well, @DanielBrito